lib/screens/transfer/send/send_enter_data/interactor/mappers/send_transaction_mapper.dart

```diff
@@ -16,10 +17,22 @@ class SendTransactionMapper extends StateMapper {
       return currentState.copyWith(pageState: PageState.failure, errorMessage: result.asError!.error.toString());
     } else {
       final resultResponse = result.asValue!.value as SendTransactionResponse;
+
+      final int currentDate = DateTime.now().millisecondsSinceEpoch;
+      bool _shouldShowInAppReview = shouldShowInAppReview;
```

Comment:
all this logic is repeated. 
This logic can be moved to a useCase and reused.

Reply:
you should not access services from a state mapper, move this to a use case

Reply:
cant have it on an usecase.

Reply:
can be either mapper or the bloc